admin/frontend/.env.example

```
@@ -0,0 +1,12 @@
+# Admin Frontend Configuration
+
+# Backend API base URL
+# Development: http://localhost:8000
+# Production: Your backend URL
+VITE_API_BASE_URL=http://localhost:8000
+
+# Base URL for the frontend application
+# This should match the path where your admin frontend is served
+# For development: /
+# For production behind reverse proxy: /admin/
+VITE_BASE_URL=/
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)